### ISSUE TITLE
Command bar height not matching Fluent React

### ISSUE THREAD

**Issue by gabbsmo** [n8design/htwoo#117]
**Describe the bug**
The height of the command bar is 46px or 48px, with `.has-overflow`.

**To Reproduce**
1. Go to [Pattern Lab](https://lab.n8d.studio/htwoo/htwoo-core/?p=molecules-cmdbar)
2. Inspect the height of `.hoo-cmdbar`

**Expected behavior**
As in [Fluent React](https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar), where it always is 44px.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Edge
 - Version: 122

**Additional context**
.hoo-cmdbar has a 1px border transparent border, increasing the height to 46px. With `.has-overflow`, a slightly bigger `min-height` is also set, increasing the hight to 48px.

**Comment by gabbsmo** [comment]
My current workaround:
```
.hoo-cmdbar {
    border: none;
}

.hoo-cmdbar.has-overflow {
    min-height: inherit;
}
```

**Comment by StfBauer** [comment]
🛳️ Shipped in 2.5.0 - https://lab.n8d.studio/htwoo/changelog/htwoo-core-cl/
